src/views/listview/list-view.html

```diff
@@ -1,52 +1,64 @@
-<div class="list-group list-view-pf">
+<div class="list-group list-view-pf" dnd-list="items" ng-class="{'list-view-pf-dnd': config.dragEnabled === true}">
+  <div class='dndPlaceholder'></div>
```

Comment:
yeah, but if drag/drop is not enabled it should be OK without it right?
(ng-if="config.dragEnabled")